imagetest/fixtures.go

```diff
@@ -28,6 +29,7 @@ const (
 	createNetworkStepName    = "create-networks"
 	createSubnetworkStepName = "create-sub-networks"
 	successMatch             = "FINISHED-TEST"
+	localShutdownScriptPath  = "/shutdown_script.txt"
```

Comment:
cant use a constant here, it will get clobbered. needs to be a uuid or the 3-tuple of testname, vmname, testimage